systemtests/tests/bconsole-pam/etc/bareos/bareos-dir.d/profile/operator.conf

```diff
@@ -3,7 +3,7 @@ Profile {
    Description = "Profile allowing normal Bareos operations."
 
    Command ACL = !.bvfs_clear_cache, !.exit, !.sql
-   Command ACL = !configure, !create, !delete, !purge, !prune, !sqlquery, !umount, !unmount
+   Command ACL = configure, !create, !delete, !purge, !prune, !sqlquery, !umount, !unmount
```

Comment:
You don't want to allow "configure". If you allow it, a user could create a new console with no ACL restrictions.
And if you really want to allow it, there is no need to explicitly specify it, as the last entry is "*all*", which allows all commands not explicitly denied.

Reply:
The "configure" command is needed for this test; "user3" will be added using "configure add" as requested from you.
There are no side effects to production systems because this is only used for this test, thus no concerns.